plans/fix-1880-beat-replacement.md

```
@@ -0,0 +1,126 @@
+
+> **この文書の読み方 —— 時系列のログであって、現状の仕様書ではない。**
+> 現在のコードの仕様はコードが唯一の情報源。数値は sha に紐づけて書くこと。
+
+## 症状
+
+MCP ツール `presentMulmoScript` の「ビート差し替え」（`filePath` + `beatIndex` + `beat`）が
+**何も書き換えずに成功として返る**。呼んだ側から成功と区別がつかない。
+
+## 再現（実行中のサーバに対して、2026-08-28 実測）
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)